pkg/vtpm/src/server.cpp

```diff
@@ -59,13 +59,9 @@ std::list<string> allowed_commands = {
 "tpm2_getcap",
 "tpm2_readpublic",
 "tpm2_startauthsession",
-"tpm2_policysecret",
```

Comment:
AFAIK, tpm2_policysecret is used by guest library, so this should be retained:
https://github.com/lf-edge/eve-tools/blob/master/eve-tools/lib/include/eve_tpm_service.h#L66

In general, what we list here should be in sync with what we support in the eve-tools library above. P

Reply:
I see that, but what is the use-case?

Reply:
yes, pls check this out:
https://github.com/lf-edge/eve-tools/blob/master/azure-on-eve/src/hsm_client_tpm_device.c#L381